src/roiextractors/extractors/simaextractor/simasegmentationextractor.py

```diff
@@ -51,6 +42,14 @@ def __init__(self, file_path: PathType, sima_segmentation_label: str = "auto_ROI
         sima_segmentation_label: str
             name of the ROIs in the dataset from which to extract all ROI info
         """
+        try:
```

Comment:
Same as other PR

Reply:
Which PR?

Reply:
He means another one in neuroconv where I did the same.

Reply:
He wants to use importlib instead of try-except